parm/use_cases/model_applications/convection_allowing_models/MODE_fcstFV3_obsGOES_BrightnessTempObjs.conf

```diff
@@ -0,0 +1,103 @@
+[config]
+# Loop by model initialization
+LOOP_BY = init
+
+INIT_TIME_FMT = %Y%m%d%H
+
+# Start time for METplus run
+INIT_BEG = 2019052100
+
+# End time for METplus run
+INIT_END = 2019052100
+
+# Increment between METplus runs in seconds. Must be >= 60
+INIT_INCREMENT = 3600
+
+LOOP_ORDER = processes
+
+PROCESS_LIST = MODE(lsm1), MODE(mp1), GridStat(lsm1), GridStat(mp1)
+
+# Sequence of leads to process
+LEAD_SEQ = 1,2
+
+FCST_IS_PROB = false
+
+# MODE variables for the forecast and observations
+MODE_QUILT = FALSE
+
+MODE_CONV_RADIUS = 5
+
+MODE_CONV_THRESH = <=235
+
+MODE_MERGE_THRESH = <=235
+
+MODE_MERGE_FLAG = NONE
+
+MODE_MASK_POLY = {PARM_BASE}/use_cases/model_applications/convection_allowing_models/MODE_fcstFV3_obsGOES_BrightnessTempObjs/CentUS.nc

```

Comment:
This file should be added to the sample input data and referenced from there instead of being put in the repository. It is too big!

Reply:
How do I add a mask file to the sample data?  The same way that I would add a data tarball?

Reply:
You add it to the directory on mohawk. There are steps to follow in the Contributor's Guide. I can take care of this for you now. The disk on mohawk is at 100% so some of the instructions won't work as expected.

Reply:
Okay.  Thanks